apps/builder/app/shared/copy-paste/plugin-instance.ts

```diff
@@ -143,6 +143,22 @@ const getAssetsUsedInProps = (props: Prop[], foundAssetsIds = new Set()) => {
   return foundAssetsIds;
 };
 
+const getPropTypeAndValue = (value: unknown) => {
```

Comment:
why do we have this function twice?

Reply:
It is temporary. Will be gone once we add ui stuff etc